tests/cephfs/cephfs_utils.py

```diff
@@ -860,12 +910,12 @@ def file_locking(self, clients, mounting_dir):
 try:
     f = open('%sto_test_file_lock', 'w+')
     fcntl.lockf(f, fcntl.LOCK_EX | fcntl.LOCK_NB)
-    print "locking file:--------------------------------"
+    print("locking file:--------------------------------")
```

Comment:
use of `log` makes sense than `print` and the same where ever

Reply:
Hi Rakesh,
print statement was written to a python file.
This python file is copied to the node and we are executing it on the node, so we haven't used log.info